### PR TITLE
Support for connectivity tol. in .par file and changes to use `next` branch of parRSB

### PR DESCRIPTION
**Pull request by thilinarmtb**
@stgeke: I will send a similar PR for nekRS.

### CODE REVIEW

File: core/map2.f [hunk 1]
```diff
@@ -183,8 +185,12 @@ subroutine get_vert_map(nlv)
 
       if (ierr.ne.0) then
         ifread_con = .false.
-        call find_con(wk,nwk,0.2,ierr)
-        if(ierr.ne.0) call find_con(wk,nwk,0.01,ierr)
+        tol = connectivityTol
+        call find_con(wk,nwk,tol,ierr)
+        if(ierr.ne.0) then
+          tol = tol / 2.0;
```
Comment (stgeke) [inline core/map2.f]: I think we should reduce it by 10x

Comment (thilinarmtb) [reply core/map2.f]: Done.

**Comment by thilinarmtb** [comment]
@stgeke: If there is no .par file this makes `connectivityTol` to 0. I guess I need to set defaults for when a `.rea` is present.

**Comment by stgeke** [comment]
Included in https://github.com/Nek5000/Nek5000/commit/8c8f781400e05d4151f573b2e67cb30ccfb38a50

**Pull request closed without merging**